doc/tutorials/using_xenomai_as_uos.rst

```diff
@@ -0,0 +1,81 @@
+.. _using_xenomai_as_uos:
+
+Using Xenomai as User OS
```

Comment:
How about adjusting the title to mention we run this as a Real Time Virtual Machine (RTVM)?
```suggestion
Using Xenomai as User OS (Real-Time VM)
```